### PR TITLE
Dynamic Hamburger menu

### PR DESCRIPTION
**Pull request by algerina**
In this Pull request I implemented the following features using Javascript:

- When the user clicks the hamburger button,  I added a mobile menu that should appear.
 - When the user clicks the close (X) button, the mobile menu disappears.
 - When the user clicks any of the mobile menu options, the mobile menu disappears.
- When the user clicks any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -87,15 +84,54 @@ body {
...
+}
+
+.hambu-menu {
+  display: none;
+  background-color: rgba(96, 112, 255, 1);
+  flex-direction: column;
+  position: fixed;
+  height: 100vh;
+  top: 0;
+  width: 557px;
```
Comment (dasileker) [inline style.css:106]: - [ ] Please make the width fit the page.

File: style.css [hunk 2]
```diff
@@ -87,15 +84,54 @@ body {
...
+  gap: 50px;
+}
+
+.hambu-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+  position: relative;
+  right: 169px;
```
Comment (dasileker) [inline style.css:117]: - [ ] Please use best practice CSS, instead `right: 169px` you can use `text-align: center;`

File: style.css [hunk 3]
```diff
@@ -87,15 +84,54 @@ body {
...
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: white;
+  text-decoration: none;
+  text-align: left;
+}
+
+.x-img {
+  right: -10%;
```
Comment (dasileker) [inline style.css:131]: - [ ] Please adjust the X icon to be more good looking like -35px

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @algerina** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **To Highlight**

- Linters are passing.
- Commit are good.
- The PR is nice descriptive.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (algerina) [reply style.css:106]: @dasileker Thank you for your valuable reviews, the width for me is fitting well can you explain more, please.

File: index.html [hunk 4]
```diff
@@ -3,45 +3,65 @@
...
+        <div id="menu-toggle" class="flex-menu"><img src="./img/Menu-button.png" alt="menu"></div>
         <ul class="desktop-toolbar">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
         </ul>
       </nav>
```
Comment (SajjadAhmad14) [inline index.html:34]: [ OPTIONAL ]The hamburger menu is aligned left on the tablet viewport. It should cover the full-screen width and follow the design. Kindly see the screenshot below and fix the width issue.
tablet

Comment (SajjadAhmad14) [inline index.html:34]: [ OPTIONAL ] Please add unnecessary files and folders like `vscode` to .gitignore.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Amel Khiri, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**